examples/plot_credit_loan_decisions.py

```diff
@@ -159,7 +159,7 @@
 dataset.head()
 
 # %%
-# From the `dataset description :footcite:`yeh2009comparisons`,
+# From the `dataset description` :footcite:`yeh2009comparisons`,
```

Comment:
Hi @emma-carballal, I wonder if we need the ticks on the `dataset description` at all. Removing both would also solve the issue, as long as one is not trailing and thus breaking the footnote link, and it seems that that is the most common style with all the others above. Do you mind checking the difference in rendering in that situation and sharing a screenshot?